test/functional/apps/dashboard_elements/controls/options_list.ts

```diff
@@ -427,6 +507,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           await dashboardControls.optionsListOpenPopover(controlId);
           await dashboardControls.optionsListPopoverSelectOption('B');
           await dashboardControls.optionsListEnsurePopoverIsClosed(controlId);
+          await dashboard.waitForRenderComplete();
```

Comment:
This test was **ever so slightly** flaky - literally failed a single time in about 200 iterations. Putting this here made it so that I was able to get it to pass 100 times without failure - hopefully that means it's no longer flaky? Hard to know cause it's so rare 🤷

Reply:
Awesome! Just a reminder to merge from upstream and unskip this one!